src/lib.rs

```diff
@@ -1,12 +1,8 @@
 #![recursion_limit = "128"]
 #[macro_use]
 extern crate log;
-extern crate onig;
 #[macro_use]
 extern crate serde;
-extern crate serde_json;
```

Comment:
is this line intentionally removed? it doesn't look related to any of the other changes

Reply:
I ended up removing this since the extern crate declaration isn't necessary anymore with the 2018 edition, and I was moving the `lazy_static`/`onig` declarations.